spec/query_spec.rb

```diff
@@ -123,6 +123,20 @@ def test_method
     end
   end
 
+  describe '#at' do
+    it 'performs at for given expression' do
+      instance = create_instance
+      ref = instance[:ref]
+      ts = instance[:ts]
+
+      prev_ts = ts - 1
+      value = random_number
+      client.query { insert(ref, prev_ts, :create, data: { x: value }) }
+
+      expect(client.query { at(Fauna.time_from_usecs(prev_ts), get(ref)) }[:data]).to eq(x: value)
```

Comment:
Oh, huh, I assumed it couldn't take one, but it can. Even if it doesn't, I should have used epoch I think. Will fix.